unit-tests/FormsTest.php

```diff
@@ -540,4 +540,13 @@ public function testElementAppendMessage()
 		$element = new Select('test-select');
 		$element->appendMessage(new Message(''));
 	}
+
+	public function testFormHasField()
```

Comment:
There is no need for this method here, in future `unit-tests` folder will be removed in favor of `tests`